backdoor-ctf-2015/binary/echo/README.md

```diff
@@ -17,4 +17,4 @@
 
 ## Other write-ups and resources
 
-* none yet
\ No newline at end of file
+* https://shankaraman.wordpress.com/2015/04/02/backdoor-ctf-2015-echo-writeup/
```

Comment:
it would have been better if you have enclosed the link with <> the standard markdown format